core/post.ts

```diff
@@ -94,6 +104,73 @@ export function getPostIntro(post: OriginPost): string {
   }
 }
 
+export async function calcPostReading(
+  post: OriginPost,
+  feed?: FeedValue,
+  filterId?: string
+): Promise<'fast' | 'slow'> {
+  let checker = prepareFilters(
+    await loadFilters({
```

Comment:
Loading filters again for every posts in feed can be very slow.

For instance, in `changePostsByFeed` you can prepare `checker` once and use it for all posts.